tests/Unit/inc/Addon/Varnish/Subscriber/cleanHome.php

```diff
@@ -16,6 +16,12 @@
  */
 class Test_CleanHome extends TestCase {
 
+	public static function setUpBeforeClass() {
+		parent::setUpBeforeClass();
+
+		require_once WP_ROCKET_PLUGIN_ROOT . 'inc/functions/i18n.php';
```

Comment:
@hellofromtonya may I ask you why you added require_once in here? Will this be used in this Unit test when we will revisit this code?

Reply:
@crystinutzaa Multiple test class files are loading this same file. With `require_once`, PHP checks if the file has already been loaded into memory. If yes, it doesn't load it again.